ps_mbo.php

```diff
@@ -77,6 +79,7 @@ class ps_mbo extends Module
         'dashboardZoneOne',
         'dashboardZoneTwo',
         'dashboardZoneThree',
+        'actionPresentModule',
```

Comment:
we will need some refacto afterwards
This PR https://github.com/PrestaShopCorp/ps_mbo/pull/152/files removes this hook call